src/main/java/io/appform/dropwizard/actors/base/UnmanagedPublisher.java

```diff
@@ -42,6 +44,7 @@ public UnmanagedPublisher(
         this.connection = connection;
         this.mapper = mapper;
         this.queueName = NamingUtils.queueName(config.getPrefix(), name);
+        this.observer = connection.getRootObserver();
```

Comment:
How are we chaining the observers if we are just calling the root observer here?